pylint/test/extensions/test_check_raise_docs.py

```diff
@@ -115,10 +115,14 @@ def my_func(self):
             """This is a docstring.
 
             :raises RuntimeError: Always
-            :raises NameError: Never
+            :except NameError: Never
```

Comment:
Ah, got it. Fixed the test and reverted my changes to possible_exc_types. Thanks!